<?xml version="1.0" encoding="UTF-8" ?>
<testsuite errors="0" failures="1" hostname="pcle" name="restaurantetocomfome.PedidoSpec" tests="1" time="3.885" timestamp="2015-10-24T02:04:06">
  <properties />
  <testcase classname="restaurantetocomfome.PedidoSpec" name="test something" time="3.813">
    <failure message="Condition not satisfied:&#xa;&#xa;Permissao.count() == 1&#xa;          |       |&#xa;          2       false&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Condition not satisfied:

Permissao.count() == 1
          |       |
          2       false

	at restaurantetocomfome.PedidoSpec.test something(PedidoSpec.groovy:29)
</failure>
  </testcase>
  <system-out><![CDATA[--Output from test something--
]]></system-out>
  <system-err><![CDATA[--Output from test something--
]]></system-err>
</testsuite>
